server/domain/status/Status.js

```diff
@@ -158,7 +160,7 @@ class Status {
     }
 
     getKey() {
-        return this.data.key;
+        return this.data.key.toLowerCase();
```

Comment:
This will cause an object to have an ugly key, but only return it "pretty". Can we make a function `filterStatus()` where we return a mutated object (aka rawStatus) with the filtered/updated fields? And throw that into the validator.

Reply:
Yes, as discussed I am introducing a statusFactory that does all the heavy lifting for us :)